tests/sparseml/transformers/obcq/test_consecutive_runs.py

```diff
@@ -114,7 +114,7 @@ def setUp(self):
         self.output_second = Path(self.output) / "test_2"
 
     def test_consecutive_runs_small(self):
-        self._test_consecutive_runs(tolerance=1e-3)
+        self._test_consecutive_runs(tolerance=1e-1)
```

Comment:
Why did we have to up the tolerance back?

Reply:
This PR was where we originally found the issue and upped the tolerance, our PR come after this and fix this tolerance, this will be easier to review as a combined unit in our feature branch